src/Componets/SelectCategory/__test__/SelectCattegory.test.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import {
+  fireEvent,
+  render, screen,
+} from '@testing-library/react';
+import Select from '../SelectCategory';
+
+describe('NameInput', () => {
+  it('should render and have placeholder', () => {
+    render(<Select
+      value=""
+      onChange={() => {}}
+      selected={[]}
+      options={['dog', 'cats']}
```

Comment:
move `['dog', 'cats']` to the top of the file and use it as a const `const mockedOptions = ['dog', 'cats']` in tests
`options={mockedOptions}`